dcpy/connectors/edm/publishing.py

```diff
@@ -0,0 +1,42 @@
+from datetime import date
+from pathlib import Path
+
+from dcpy.connectors import s3
+from dcpy.git import git_branch
+
+BUCKET = "edm-publishing"
+
+
+def upload(
+    output: Path,
+    publishing_folder: str,
+    acl: str,
+    *,
+    s3_subpath: str = None,
+    latest: bool = True,
+    include_foldername: bool = False,
+    max_files: int = 20
+):
+    if s3_subpath is None:
+        ## potential TODO, change this to environment variable
+        s3_subpath = git_branch()
```

Comment:
This feels misplaced here. Would it be possible to have the callers of this method figure out the path / git_branch, and just ensure that the `s3_subpath` isn't null? Would help us toward separation of concerns with edm and git

Reply:
agree it'd be great to separate S3 from git. so maybe no default value for `s3_subpath`?